tests/saw/spec/extras/HMAC/README

```diff
@@ -0,0 +1,18 @@
+These are files detailing the verification of HMAC with respect to the
+HMAC specification provided by Andrew Appel's HMAC verification effort.
+
+These files don't build by themselves, but rather require the
+cryptol-semantics repository
+(https://github.com/GaloisInc/cryptol-semantics) checked out and built
+in the same directory as s2n.
+
+These is verification effort used Coq 8.6, if you use a different
+version YMMV.
+
+Build instructions:
+  1. Build the cryptol-semantics repository
+  2. make (in this directory)
+
```

Comment:
This directory has a lot of `*.v` files. I'm assuming that these are [VST](https://github.com/PrincetonUniversity/VST) Files? Can you add a few links for "Getting Started" resources for these files to this README?